### DISCUSSION TITLE
Should we move the Enhancements section to the unversioned documentation?

### DISCUSSION THREAD

**Discussion by Abhijay007** [uptane/uptane.github.io#141] [General]
We recently moved the "learn more" section to the unversioned part of the documentation, and as @hexsecs  suggested previously, we should only version the "Standard" and "Deployment Best Practices" sections. Currently, we have separate versions for the "Enhancements" sections. Philip and I discussed this during the last website call, but we've decided to seek input from other Uptane community members and contributors before making any final decisions.

**Comment by hexsecs** [comment]
As discussed in today's, call we will unversion the enhancements page.
